backend/src/test/resources/application-test.yml

```
@@ -3,3 +3,9 @@ logging:
     org.springframework.web.socket: DEBUG
     org.springframework.messaging: DEBUG
     coffeeshout: DEBUG
+
+room:
+  cleanup:
+    interval: 500ms # 500밀리초 (테스트용)
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)